src/App.js

```diff
@@ -1,15 +1,31 @@
 import React, { useState } from 'react'
 import Form from './components/Form'
-import Card from './components/Card/Card'
+import Card from './components/Card/Cards'
```

Comment:
Rename Card into cards just like the name of the component, because the component gives you a set of 'cards' filtered by different categories.